journal-of-the-royal-society-of-western-australia.csl

```diff
@@ -203,6 +203,7 @@
     <sort>
       <key macro="author"/>
       <key variable="title"/>
+      <key macro="year"/>
```

Comment:
This should at least come before the "title" sort, no?

Reply:
cheers. Fixed.